credit_test1/task1/main.c

```diff
@@ -0,0 +1,33 @@
+#include <stdio.h>
+#include <string.h>
+
+#define BUFFER_SIZE 257
+#define BASE 3
+
+int main() {
+    char buffer[BUFFER_SIZE];
+    printf("enter the string:\n");
+    scanf("%256s", buffer);
+    int length = strlen(buffer);
+    int start = 0;
+    while (start < length && buffer[start] == '0')
+        start++;
+    if (start == length) {
+        printf("0\n");
+        return 0;
+    }
+    if ((length - start) % BASE == 1) {
+        printf("%d", buffer[start] - '0');
```

Comment:
```suggestion
        printf("%с", buffer[start]);
```
Уже получили символ, который можно печатать. Так зачем его в число превращать? Трижды в коде
(-0.5)